controllers/imagecollector/imagecollector_controller.go

```diff
@@ -233,14 +233,14 @@ func (r *Reconciler) createImageJob(ctx context.Context, req ctrl.Request, argsC
 	scanDisabled := *scannerImage == ""
 	startTime = time.Now()
 
-	job := &eraserv1alpha1.ImageJob{
+	job := &eraserv1.ImageJob{
```

Comment:
let's keep this on v1alpha1 to make sure it works too

Reply:
I'm fine making this change, just want to understand why. If we're creating a new API version, why use the old version in the code?

Reply:
I think we just want a few remaining to ensure we haven't broken existing usage. I am okay to keep this in v1 though since we already have a few scattered around.